frontend/src/Editor/Inspector/EventManager.jsx

```diff
@@ -131,14 +133,24 @@ export const EventManager = ({
               <span data-cy="event-label">Event</span>
             </div>
             <div className="col-9" data-cy="event-selection">
-              <SelectSearch
+              <Select
                 className={`${darkMode ? 'select-search-dark' : 'select-search'}`}
                 options={possibleEvents}
                 value={event.eventId}
                 search={false}
                 onChange={(value) => handlerChanged(index, 'eventId', value)}
-                filterOptions={fuzzySearch}
                 placeholder="Select.."
+                styles={{
+                  ...styles,
+                  menuPortal: (provided) => ({ ...provided, zIndex: 9999 }),
+                  menuList: (base) => ({
+                    ...base,
+                    '::-webkit-scrollbar': {
+                      width: '2px',
+                    },
+                  }),
+                }}
+                useMenuPortal="true"
```

Comment:
Instead of string, `bool` value can be passed
```suggestion
                useMenuPortal={true}
```